Source/Core/Core/System.h

```diff
@@ -31,13 +31,13 @@ namespace DSP
 {
 class DSPManager;
 }
-namespace DVDInterface
+namespace DVD
 {
-class DVDInterfaceState;
+class DVDInterface;
 }
-namespace DVDThread
+namespace DVD
```

Comment:
Seems like this can now be merged with the same namespace above

Reply:
Yup, done.